lib/tool_shed/util/workflow_util.py

```diff
@@ -78,7 +78,7 @@ def get_data_inputs( self ):
         data_inputs = []
 
         def callback( input, value, prefixed_name, prefixed_label ):
-            if isinstance( input, galaxy.tools.parameters.DataToolParameter ):
+            if isinstance( input, galaxy.tools.parameters.basic.DataToolParameter ):
```

Comment:
I am -0 on this change - see note https://github.com/galaxyproject/tools-devteam/pull/155/files#r32624021. Same applies here I would be tempted to not change this. Though at least this is internal to Galaxy so I feel less anxious about digging into the internals of ``galaxy.tools.parameters`` and there are probably other places we use the basic to reference the class (you could probably find blames where I reference the parameter this way).